specification/attestation/data-plane/Microsoft.Attestation/preview/2022-09-01-preview/attestation.json

```diff
@@ -748,6 +743,10 @@
         "minimumPswWindowsVersion": {
           "description": "Minimum Windows PSW version required to support the corresponding Tcb baseline. SGX TEE specific property",
           "type": "string"
+        },
+        "isSelectedTcb": {
```

Comment:
Wasn't there to be a change to the endpoint to get _only_ the selected TCB?

Reply:
Per API review discussion, we agreed upon not having an endpoint to get only the selected TCB. Customers will go through the list and find the selected TCB.
We will create an endpoint to get selected TCB if there is customer request.